build/docker/lambda/Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM amazon/aws-lambda-python:3.9
```

Comment:
Is there any amazon/aws-lambda-python:3.10.5  to match our version?

Reply:
No, not yet Amazon doesn't support 3.10 for either runtimes or AWS Built containers. We could look into building our own images from scratch to allow upgrading to 3.10 or 3.11 when it comes out